actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -146,6 +148,12 @@ def _filtered_referrer # :nodoc:
       #   link_to nil, "http://example.com"
       #   # => <a href="http://www.example.com">http://www.example.com</a>
       #
+      # Pithier yet, when name is an ActiveRecord model that defines a
```

Comment:
I'm not sure I understand the use of "pithier" here but perhaps my English skills are deteriorating 🤔

Reply:
Pithy means concise or forcefully expressive. Something concise and full of meaning, which IMO is exactly what a model is in relation to a URL helper. 🤗

Reply:
That said we can change it to "concise" to be more ESL friendly. I'll just point out I'm the ESL learner and you're the native English speaker. 😛

Reply:
Mind you I think I just used the same adjective I found earlier in the docs: https://github.com/rails/rails/blob/61fb58f6a7f852d765d616d1da41d17851ec7afc/actionview/lib/action_view/helpers/url_helper.rb#L124